sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -2755,91 +2713,1315 @@
+                {
+                    "label": "Student did not complete program",
+                    "value": "studentDidNotCompleteProgram",
+                    "shortcut": ""
+                },
+                {
+                    "label": "Student withdrew from program",
+                    "value": "studentWithdrewFromProgram",
+                    "shortcut": ""
+                }
+            ],
+            "dataType": "",
+            "persistent": true,
+            "protected": false,
+            "dbIndex": false,
+            "encrypted": false,
+            "redrawOn": "",
+            "clearOnHide": true,
+            "customDefaultValue": "",
+            "calculateValue": "",
+            "calculateServer": false,
+            "allowCalculateOverride": false,
+            "validate": {
+                "required": true,
+                "onlyAvailableItems": false,
+                "customMessage": "",
+                "custom": "",
+                "customPrivate": false,
+                "json": "",
+                "strictDateValidation": false,
+                "multiple": false,
+                "unique": false
+            },
+            "errorLabel": "",
+            "errors": "",
+            "key": "scholasticStanding",
+            "tags": [],
+            "properties": {},
+            "conditional": {
+                "show": null,
+                "when": null,
+                "eq": "",
+                "json": ""
+            },
+            "customConditional": "",
+            "logic": [],
+            "attributes": {},
+            "overlay": {
+                "style": "",
+                "page": "",
+                "left": "",
+                "top": "",
+                "width": "",
+                "height": ""
+            },
+            "type": "radio",
+            "input": true,
+            "placeholder": "",
+            "prefix": "",
+            "suffix": "",
+            "multiple": false,
+            "unique": false,
+            "refreshOn": "",
+            "dataGridLabel": false,
+            "widget": null,
+            "validateOn": "change",
+            "showCharCount": false,
+            "showWordCount": false,
+            "allowMultipleMasks": false,
+            "addons": [],
+            "inputType": "radio",
+            "fieldSet": false,
+            "id": "eithvn8",
+            "defaultValue": ""
+        },
+        {
+            "title": "Student went from full-time to part-time studies (vice-versa) or started a different program Panel",
+            "labelWidth": "",
+            "labelMargin": "",
+            "theme": "default",
+            "tooltip": "",
+            "customClass": "",
+            "collapsible": false,
+            "hidden": false,
+            "hideLabel": true,
+            "disabled": false,
+            "modalEdit": false,
+            "key": "studentWentFromFullTimeToPartTimeStudiesViceVersaOrStartedADifferentProgramPanel",
```

Comment:
Can we make it shorter, please?

Reply:
this is actually the panel API name, we don't use this in our code. but i can change